### ISSUE TITLE
Preventing over fitting in reconstruction and dynamics loss

### ISSUE THREAD

**Issue by truncs** [danijar/dreamerv3#150]
When training over the long duration it seems like even though the training loss is decreasing the the open loop loss and report loss continuously increases after bottoming out (see the attached graph).

![train_report_camera_loss](https://github.com/user-attachments/assets/05ac899e-5bac-48b1-b6f8-ff6e7047ae0f)

Similar pattern was observed  for dynamics loss for *all* experiments

![dyn_loss](https://github.com/user-attachments/assets/b601af57-c572-47a1-a42b-9edb311c27ba)

It this correct to interpret that this just means that the model is over fitting to the existing data in the replay buffer while under fitting to new data and hence the model is not general enough?
I do see that the reward score is still improving (see screenshot below), but I assume that might be because of the SAC policy improving  (see actor loss screenshot below)
![loss_with_
![actor_loss](https://github.com/user-attachments/assets/908e304b-cb19-4dc2-81dd-20b98dc2d6cc)
score](https://github.com/user-attachments/assets/5701f591-e3c0-473a-a8e1-1d8e2282ba6d)

Similar behavior of divergence between train and openloop was observed in critic loss as well
![critic_loss](https://github.com/user-attachments/assets/2bc1de12-a456-47a7-a3d8-33bd0549cd90)

My questions are

1. Is this interpretation of over fitting correct?
2. Does this lead to less sample efficiency?
3. Any strategies to mitigate it?

Would love to know your thoughts on this.

**Comment by danijar** [comment]
You can choose a run script with eval metrics to diagnose overfitting. Losses going up is normal when the agent explores the environment further and sees more diverse data over time.
